initializr-service/src/main/resources/application.yml

```diff
@@ -398,6 +398,17 @@ initializr:
             - versionRange: "[1.2.0.RELEASE,1.3.0.RC1)"
               version: 1.0.1.RELEASE
           scope: test
+        - name: Spring Cloud Open Service Broker
+          id: open-service-broker
+          description: Create service brokers that conform to the Open Server Broker API specification
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-starter-open-service-broker-webmvc
```

Comment:
This is a starter that brings `spring-boot-starter-web`. In 2.0 we've tried as much as we can to avoid that as there is the reactive flavour as well.

I understand that Webflux is not supported. Is that on the roadmap? If so, I'd very much prefer that one has to chose this dependency and the web flavour they're targeting.